Tasks/NodeToolV0/nodetool.ts

```diff
@@ -21,6 +22,20 @@ async function run() {
     }
 }
 
+function getVersionSpec() {
+    const versionSpecInput = taskLib.getInput('versionSpec', true);
+
+    const cleanVersion = toolLib.cleanVersion(versionSpecInput);
+    if (cleanVersion) {
+        taskLib.debug(`version spec '${cleanVersion}' from '${versionSpecInput}'`);
+        return versionSpecInput;
+    }
+
+    const versionSpec = fs.readFileSync(versionSpecInput, { 'encoding': 'utf' });
```

Comment:
Could it make sense to add separate task input for version spec specified as file path - to avoid any confusion around it?